src/test/java/com/google/javascript/gents/multiTests/module_imports/import_binding.ts

```diff
@@ -6,6 +6,10 @@ import * as namespace from './export_namespace';
 import {D} from './export_both';
 import * as DExports from './export_both';
 import {bar} from './export_both';
+import R from 'goog:module.R';
```

Comment:
this is a bit misleading - if you don't `declareLegacyNamespace`, `"module".R` is completely different from `"module.R"`. That is, this code should fail, there shouldn't be anything providing `goog:module.R`.

Is that what you're after here? Also, the test doesn't seem to fail..

Reply:
This illustrates the incorrect behavior, which should be 
import {R} from 'goog:module';

when fixed.